sphinx/writers/latex.py

```diff
@@ -806,10 +806,8 @@ def visit_desc_parameterlist(self, node: Element) -> None:
                                        for c in node.children]
         # How many required parameters are left.
         self.required_params_left = sum(self.list_is_required_param)
-        self.param_separator = r'\sphinxparamcomma, '
+        self.param_separator = r'\sphinxparamcomma '
```

Comment:
The space token here is gobbled by TeX when gathering the `sphinxparamcomma`-named control sequence and it serves to avoid a letter following immediately if any (but that should not happen as it will be a priori either a `}` or a  `\sphinxparam`) to create an undefined LaTeX control sequence.